package.json

```diff
@@ -73,30 +73,25 @@
     "universal-cookie": "4.0.2"
   },
   "devDependencies": {
-    "@edx/frontend-build": "^2.0.6",
-    "audit-ci": "^2.5.1",
+    "@edx/frontend-build": "5.0.3",
+    "audit-ci": "2.5.1",
     "axios-mock-adapter": "1.17.0",
     "chance": "1.1.0",
-    "clean-webpack-plugin": "3.0.0",
```

Comment:
Leveraging webpack-merge allows us to eliminate re-adding plugins that already exist in our webpack config. presets and therefore this repo doesn't need to manage their versions.